### DISCUSSION TITLE
ProcessPitchBend

### DISCUSSION THREAD

**Discussion by nathanmariano-dev** [laurence-myers/midi-to-macro#4] [Q&A] [answered]
For some reason, 
`ProcessPitchBend(device, channel, value) {`
`    SendInput %value%`
`}`
doesn't output anything. 
I've replaced 'value' with the other parameters and I've received an input from them. 
The MIDIMonitor detects my pitch slider perfectly fine. I can even see the exact values of the CC there. Is there any fix to this?

Thanks for creating this!

**Comment by laurence-myers** [comment, marked as answer]
Thanks for raising this! I've just pushed up a fix, please grab the latest version of the code.

Note that pitch bend values have a range from 0 to 16,383, with a value of 8192 meaning "no bend".

Let me know if you have any further issues. 🙂

**Reply by nathanmariano-dev** [reply]
Thanks! Everything worked perfectly :D
